### ISSUE TITLE
META data link to perl.org instead to GitHub

### ISSUE THREAD

**Issue by ppisar** [Dual-Life/Devel-PPPort#69]
Because Devel-PPPort development happens on Github, it would be great to update metadata (Makefile.PL:102) to actually point a bugtracker and a repository to GitHub. It links to perl.org RT and gitweb now.

A similar change has already been made in a POD (https://github.com/Dual-Life/Devel-PPPort/issues/5).

**Comment by karenetheridge** [comment]
> Because Devel-PPPort development happens on Github

It actually doesn't. Some may find it more convenient to post pull requests here for review, but Devel-PPPort is a dual-life, *blead-first* distribution and all changes technically happen there first, and reviewed by the porters, before then being duplicated here in this repository for a cpan release.

**Comment by atoomic** [comment]
@karenetheridge I agree with your assessment concerning `dist` modules but in practice over the last few months most of the development for PPPort occurs there directly as we can easily use Travis CI.

Indeed we make sure to keep `blead` in sync with `master` and only release what is in blead!

**Comment by khwilliamson** [comment]
So, what to do about this ticket @atoomic @karenetheridge

**Comment by karenetheridge** [comment]
Even though releases occur in blead first, it would be desirable to attract bug reports and pull requests here.  Development pretty clearly happens in this repository now, not github.com/Perl/perl5, so I am in favour of changing the metadata.

**Comment by khwilliamson** [comment]
On 12/15/19 11:10 AM, Karen Etheridge wrote:
> Even though releases occur in blead first, it would be desirable to 
> attract bug reports and pull requests here. Development pretty clearly 
> happens in this repository now, not github.com/Perl/perl5, so I am in 
> favour of changing the metadata.
> 
> —

That makes sense to me.
